src/util/types.hpp

```diff
@@ -26,6 +26,9 @@ using i64 = std::int64_t;
 using i128 = __int128;
 #pragma GCC diagnostic pop
 
+using Value = i32;
+using Depth = i32;
```

Comment:
This should be i16 if anything? I doubt you'll ever hit the limit of i16.

Reply:
I am leaving it as is for now, but feel free to open an issue. I imagine we will change this once we implement a transposition table, since we may even end up using an even smaller datatype.

Reply:
I wont change this for now, as i feel it wont matter. Maybe we will change it once transposition table are added.